src/annotationLayer.js

```diff
@@ -172,16 +183,118 @@ var annotationLayer = function (args) {
     }
   };
 
+  /**
+   * Select or deselect an edit handle.
+   *
+   * @param {geo.event} evt The mouse move event.
```

Comment:
is this only applicable/callable when "The mouse move"?

Reply:
This gets called with the mouseon and mouseoff events.  Those only occur on a mouse move or when the mouse move is retriggered.  When we want to make sure touch devices are fully supported, we should trigger mouseon and mouseoff events (and mouseover and mouseout) before the touch causes a click (but that is an issue for another PR).